style.css

```diff
@@ -61,17 +64,28 @@
   transform: rotate(45deg);
   position: absolute;
   top: 45px;
-  left: -47px; 
+  left: -47px;
 }
 
-.difficult:nth-child(1) .difficult-check{
+.difficult:nth-child(1) .difficult-check {
   top: 23px;
 }
 
 .cards {
   display: flex;
   padding-top: 21px;
   margin-right: 58px;
+  position: absolute;
+  right: 0;
+  bottom: 0;
+}
+
+.card-wrap {
+    width: 100%;
```

Comment:
Codacy found an issue: [Expected indentation of 4 spaces (indentation)](https://app.codacy.com/gh/WebHeroSchool/jsFinalTask/pullRequest?prid=6141138)